types.go

```diff
@@ -26,8 +26,14 @@ type DestinationConfig struct {
 	ExtraJSON json.RawMessage
 }
 
+const (
+	VerbosityLow  = 0
+	VerbosityHigh = 10
```

Comment:
Did you mean to use these somewhere?

Reply:
Originally, but I decided to leave them to document that 0 is low verbosity and 10 is high.